### ISSUE TITLE
Export to har file

### ISSUE THREAD

**Issue by pfeiferj** [Orange-OpenSource/hurl#1145]
### Problem to solve

Allow other tools to read information about the requests and responses that occurred during a run of hurl.

### Proposal

Add an option to hurl to export the request and response information to a har file. The har file format is a common format used by tools to do things like create mock servers, visualize network request timing, and create request runners. Using har would therefore create a large amount of interoperability with existing software.

**Comment by jcamiel** [comment]
Hi,
Hurl supports a structured JSON output with   [`--json` option](https://hurl.dev/docs/manual.html#json). The format is loosely based on HAR but we could also add an explicit HAR output that is strictly HAR.

**Comment by pfeiferj** [comment]
Hmm, i might just be able to write a little script to convert the current json output into har. But i think having built in support for it would still be ideal, the better the interoperability of hurl is the easier it is to convince people to use it

**Comment by jcamiel** [comment]
Yes yes I agree. Both options are useful, `--json` and `--har`. We want to have asserts and captures in export for instance so a custom export, different from HAR, is needed. And we can also have an explicit HAR export.
